shell/platform/darwin/ios/framework/Source/FlutterKeyboardManagerTest.mm

```diff
@@ -17,364 +15,253 @@
-  [self.mockEngine stopMocking];
-  self.mockEngine = nil;
-  [super tearDown];
-}
-
-- (id)checkKeyDownEvent:(UIKeyboardHIDUsage)keyCode API_AVAILABLE(ios(13.4)) {
-  return [OCMArg checkWithBlock:^BOOL(id value) {
-    if (![value isKindOfClass:[FlutterUIPressProxy class]]) {
-      return NO;
-    }
-    FlutterUIPressProxy* press = value;
-    return press.key.keyCode == keyCode;
-  }];
-}
-
-- (id<FlutterKeyPrimaryResponder>)mockPrimaryResponder:(KeyCallbackSetter)callbackSetter
-    API_AVAILABLE(ios(13.4)) {
-  id<FlutterKeyPrimaryResponder> mock =
-      OCMStrictProtocolMock(@protocol(FlutterKeyPrimaryResponder));
-  OCMStub([mock handlePress:[OCMArg any] callback:[OCMArg any]])
-      .andDo((^(NSInvocation* invocation) {
-        FlutterUIPressProxy* press;
-        FlutterAsyncKeyCallback callback;
-        [invocation getArgument:&press atIndex:2];
-        [invocation getArgument:&callback atIndex:3];
-        CFRunLoopPerformBlock(CFRunLoopGetCurrent(),
-                              fml::MessageLoopDarwin::kMessageLoopCFRunLoopMode, ^() {
-                                callbackSetter(press, callback);
-                              });
-      }));
-  return mock;
-}
-
-- (id<FlutterKeySecondaryResponder>)mockSecondaryResponder:(BoolGetter)resultGetter
-    API_AVAILABLE(ios(13.4)) {
-  id<FlutterKeySecondaryResponder> mock =
-      OCMStrictProtocolMock(@protocol(FlutterKeySecondaryResponder));
-  OCMStub([mock handlePress:[OCMArg any]]).andDo((^(NSInvocation* invocation) {
-    BOOL result = resultGetter();
-    [invocation setReturnValue:&result];
-  }));
-  return mock;
-}
-
-- (FlutterViewController*)mockOwnerWithPressesBeginOnlyNext API_AVAILABLE(ios(13.4)) {
+- (void)testNextResponderShouldThrowOnPressesEnded {
   // The nextResponder is a strict mock and hasn't stubbed pressesEnded.
   // An error will be thrown on pressesEnded.
   UIResponder* nextResponder = OCMStrictClassMock([UIResponder class]);
-  OCMStub([nextResponder pressesBegan:[OCMArg any] withEvent:[OCMArg any]]).andDo(nil);
+  OCMStub([nextResponder pressesBegan:OCMOCK_ANY withEvent:OCMOCK_ANY]);
 
-  FlutterViewController* viewController =
-      [[FlutterViewController alloc] initWithEngine:self.mockEngine nibName:nil bundle:nil];
+  id mockEngine = OCMClassMock([FlutterEngine class]);
+  FlutterViewController* viewController = [[FlutterViewController alloc] initWithEngine:mockEngine
+                                                                                nibName:nil
+                                                                                 bundle:nil];
   FlutterViewController* owner = OCMPartialMock(viewController);
   OCMStub([owner nextResponder]).andReturn(nextResponder);
-  return owner;
-}
 
-// Verify that the nextResponder returned from mockOwnerWithPressesBeginOnlyNext()
-// throws exception when pressesEnded is called.
-- (bool)testNextResponderShouldThrowOnPressesEnded API_AVAILABLE(ios(13.4)) {
```

Comment:
This wasn't actually running because the return type wasn't void.

Reply:
Maybe we need a custom CI check on this? It seems like anything in `Test.{m,mm}` that starts `- (anything but void)test` is probably wrong.

Reply:
Filed https://github.com/flutter/flutter/issues/146671